internal/loader/loader.go

```diff
@@ -13,28 +13,30 @@ import (
 func LoadCustomReplacementMap(path string) (map[string]string, error) {
 	customReplacementMap := make(map[string]string)
 
-	file, err := os.OpenFile(path, os.O_RDONLY, os.ModePerm)
-	if err != nil {
-		return nil, err
-	}
-	defer file.Close()
-
-	scanner := bufio.NewScanner(file)
-	for scanner.Scan() {
-		line := scanner.Text()
-		values := strings.Split(line, ":")
-		if len(values) == 1 {
-			slog.Error("wrong custom mapping: %s", "error", line)
+	if path != "" {
+		file, err := os.OpenFile(path, os.O_RDONLY, os.ModePerm)
+		if err != nil {
+			return nil, err
 		}
+		defer file.Close()
 
-		originalValue := values[0]
-		newValue := values[1]
+		scanner := bufio.NewScanner(file)
+		for scanner.Scan() {
+			line := scanner.Text()
+			values := strings.Split(line, ":")
+			if len(values) == 1 {
+				slog.Error("wrong custom mapping: %s", "error", line)
```

Comment:
As we are (currently) using slog, but in future it could be another logging library, consider please to create Logger as a dependency and is it, instead of direct call of library.